sources/install/package_ad.sh

```diff
@@ -77,12 +77,12 @@ function install_crackmapexec() {
     # Source bc cme needs cargo PATH (rustc) -> aardwolf dep
     # TODO: Optimize so that the PATH is always up to date
     source /root/.zshrc || true
-    git -C /opt/tools/ clone https://github.com/Porchetta-Industries/CrackMapExec.git
+    git -C /opt/tools/ clone https://github.com/mpgn/CrackMapExec.git
     python3 -m pipx install /opt/tools/CrackMapExec/
     add-aliases crackmapexec
     add-history crackmapexec
     add-test-command "crackmapexec --help"
-    add-to-list "crackmapexec,https://github.com/byt3bl33d3r/CrackMapExec,Network scanner."
+    add-to-list "crackmapexec,https://github.com/mpgn/CrackMapExec,Network scanner."
 }
 
 function configure_crackmapexec() {
```

Comment:
How about the comment at line 94 which still mentions byt3bl33d3r ?

```
function configure_crackmapexec() {
    colorecho "Configure crackmapexec"
    mkdir -p ~/.cme
    [ -f ~/.cme/cme.conf ] && mv ~/.cme/cme.conf ~/.cme/cme.conf.bak
    cp -v /root/sources/assets/crackmapexec/cme.conf ~/.cme/cme.conf
    # below is for having the ability to check the source code when working with modules and so on
    # git -C /opt/tools/ clone https://github.com/byt3bl33d3r/CrackMapExec
    cp -v /root/sources/assets/grc/conf.cme /usr/share/grc/conf.cme
}
```

Reply:
I didn't change because it was commented and not used anymore but yeah it need to be changed if used :)